bulldozer/config_fetcher.go

```diff
@@ -19,7 +19,7 @@ import (
 	"fmt"
 	"net/http"
 
-	"github.com/google/go-github/v32/github"
+	"github.com/google/go-github/v38/github"
```

Comment:
In https://github.com/palantir/policy-bot/pull/261 we discovered some weird compatibility with older verisons of GHE. It might be worthwhile calling that out in the README.

Reply:
I think we're safe here as those compat issues were down to the preview headers for Checks API, which Bulldozer doesn't use.

Reply:
Bulldozer reads from the Checks API to make sure checks are included for when checking required statuses against the actual statuses: https://github.com/palantir/bulldozer/blob/develop/pull/github_context.go#L238